soco/core.py

```diff
@@ -1261,28 +1261,36 @@ def get_sonos_playlists(self, *args, **kwargs):
                                                                 **kwargs)
 
     @only_on_master
-    def add_uri_to_queue(self, uri):
+    def add_uri_to_queue(self, uri, position=0, as_next=False):
         """Adds the URI to the queue.
```

Comment:
While it doesn't pertain to this PR at all, could I convince you to convert the docstring to Napoleon syntax and also to exchange the bulk of the two, so that the main (full) docstring is in add_to_queue and the docstring to add_uri_to_queue then refers to the other one.